### PR TITLE
Fixed undefined value of reconnectTries and reconnectInterval

### PR DESCRIPTION
**Pull request by Gauravp-NEC**
This PR is related to PR:https://github.com/telefonicaid/fiware-sth-comet/pull/565. While debugging it is found that the values of `reconnectTries` and `reconnectInterval` is "undefined" in MongoClient.connect( ) (https://github.com/telefonicaid/fiware-sth-comet/blob/master/lib/database/sthDatabase.js#L110-L111).

The `reconnectTries` and `reconnectInterval` parameters where missed here: https://github.com/telefonicaid/fiware-sth-comet/blob/master/lib/sth.js#L97-L105.

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -11,3 +11,4 @@
     -  Fix or disable eslint errors
 - Upgrade logops dep from 2.1.0 to 2.1.2 due to colors dependency corruption
 - Add: graceful shutdown listening to SIGTERM and SIGHUP signals (#576)
+- Fixed undefined value of reconnectTries and reconnectInterval
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:14]:
```suggestion
```

As this PR is a fix of the already existing:

```
- Add config (reconnectTries, reconnectInterval) to allow mongo driver reconnect (#559)
```

**Review by AlvaroVega — APPROVED** [review]
LGTM

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (Gauravp-NEC) [reply CHANGES_NEXT_RELEASE]: @fgalan , Done in commit https://github.com/telefonicaid/fiware-sth-comet/pull/581/commits/72bd8432cbb0226a6c27da943518f08f94c9e21f

**Review by fgalan — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**